src/__test__/DetailsPage.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import DetailsPage from '../components/DetailsPage';
+import store from '../redux/configStore';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
- [ ] Let's add more test cases in your project, currently, you have only two cases.